### DISCUSSION TITLE
Allow packages to have their own LivewireComponentsFinder

### DISCUSSION THREAD

**Discussion by sbrow** [livewire/livewire#3914] [3. Feature ideas]
As I understand it, when developing a custom package, each Livewire component in said package needs to be individually registered with `Livewire::component()`. This isn't a big deal when publishing a few components, but becomes unwieldy quite quickly.

Would it be possible to modify Livewire such that custom packages can register their own `LivewireComponentsFinder` instead? Either extending the main manifest, or if necessary using their own separate file.

Cheers.

**Comment by joshhanley** [comment]
@sbrow yeah I think it would be handy to be able specify a directory in a package for Livewire to look in. PR's are welcome 🙂
